static/index-template.html

```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>
     <%= htmlWebpackPlugin.options.title %>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
```

Comment:
a link is not a valid child of the title, it must be in the head

Reply:
removed from the title and placed within the head